src/frontend/src/handler/mod.rs

```diff
@@ -455,6 +457,32 @@ pub async fn handle(
             )
             .await
         }
+        Statement::AlterDatabase {
+            name,
+            operation: AlterDatabaseOperation::ChangeOwner { new_owner_name },
+        } => {
+            alter_owner::handle_alter_owner(
+                handler_args,
+                name,
+                new_owner_name,
+                Entity::DatabaseId(0),
```

Comment:
It's a little bit weird to use a dummy entity as the parameter for `handle_alter_owner`. How about use `StatementType` instead and infer entity inside the function?